### ISSUE TITLE
can't detect far faces on custom model

### ISSUE THREAD

**Issue by raccococococo** [dusty-nv/jetson-inference#1881]
Hi, I'm new to the world of Jetson. I'm trying to create a custom model, and when I run the program "detectnet.py" with "--network=facedetect", I can see faces from 4-5 meters away from the camera. However, with my custom model, I can only see faces at 0.5 meters from the camera. Is this problem due to the minimal resolution of 300x300 in the model "mobilenet-v1-ssd-mp-0_675.pth"? Is there a model for training that uses photos at 80x80 or 40x40?

**Comment by dusty-nv** [comment]
@raccococococo yes that is typically an issue with the model resolution, and SSD-Mobilenet can be trained at higher resolution (like 512x512), like here:

https://forums.developer.nvidia.com/t/how-train-jetson-inference-ssd512-model/168510/13?u=dusty_nv

(note that you no longer need to use different `res512` branch, that was merged into master)

**Comment by raccococococo** [comment]
Can't I use another model like ResNet18 for this training? I have read that it can be used with 80x80 images

**Comment by raccococococo** [comment]
I have tried using MobileNet V1 with 512x512 images and can always see faces at 2 meters. I need to see faces from farther away. The camera captures 2K images, so faces are seen well even from a distance, but the model can't keep up. Does anyone know of models that work with lower resolution?

**Comment by raccococococo** [comment]
@dusty-nv any idea?
